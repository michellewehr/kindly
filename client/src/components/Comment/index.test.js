import { render, cleanup, fireEvent } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import CommentList from './index';

afterEach(cleanup);

describe('Comment component renders', () => {

   it('renders', () => {
      render(<CommentList />);
   });

   it('Matches snapshot DOM node structure', () => {
      const { asFragment } = render(<CommentList />);
      expect(asFragment()).toMatchSnapshot();
   });
});