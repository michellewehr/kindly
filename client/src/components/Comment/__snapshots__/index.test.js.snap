// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment component renders Matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="text-gray-600 body-font overflow-hidden"
  >
    <div
      class="container px-5 py-24 mx-auto"
    >
      <div
        class="-my-8 divide-y-2 divide-gray-100"
      >
        <div
          class="py-8 flex flex-wrap md:flex-nowrap"
        >
          <div
            class="md:w-64 md:mb-0 mb-6 flex-shrink-0 flex flex-col"
          >
            <span
              class="font-semibold title-font text-gray-700"
            >
              //! get users name
            </span>
            <span
              class="mt-1 text-gray-500 text-sm"
            >
              //! get comment dateTime
            </span>
          </div>
          <div
            class="md:flex-grow"
          >
            <p
              class="leading-relaxed"
            >
              //! get commentText Glossier echo park pug, church-key sartorial biodiesel vexillologist pop-up snackwave ramps cornhole. Marfa 3 wolf moon party messenger bag selfies, poke vaporware kombucha lumbersexual pork belly polaroid hoodie portland craft beer.
            </p>
            <a
              class="text-indigo-500 inline-flex items-center mt-4"
            >
              //! get likeCount Like
              <svg
                class="w-4 h-4 ml-2"
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
              >
                <path
                  d="M5 12h14"
                />
                <path
                  d="M12 5l7 7-7 7"
                />
              </svg>
            </a>
          </div>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
